web/forms.py

```
@@ -1787,3 +1792,97 @@ class StudyGroupForm(forms.ModelForm):
+        return price
+
+    def clean(self) -> dict:
+        cleaned_data = super().clean()
+        price_monthly = cleaned_data.get("price_monthly")
+        price_yearly = cleaned_data.get("price_yearly")
+
+        if price_monthly and price_yearly:
+            yearly_equivalent = price_monthly * 12
+            if price_yearly >= yearly_equivalent:
+                self.add_error(
+                    "price_yearly", "Yearly price should offer a discount compared to paying monthly for 12 months."
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)